0001-typedb/02-dev/02-schema.md

```diff
@@ -0,0 +1,948 @@
+
+To define a new entity type that owns one or more attribute types, use the `owns` keyword followed by the name of the 
+attribute type. The attribute types are appended to the entity type definition with commas. Note, attribute types must 
+be defined before or concurrently (in the same transaction) with its owner(s). You can add owners later but you can't 
+own non-existent attribute type.
+
+<!-- test-ignore -->
+```typeql
+define
+
+object-type sub attribute, value string;
+object sub entity, abstract, owns object-type;
+```
+
+###### Cardinality
+
+By default, an entity can have multiple attributes of the same type: zero, one, or many.
+
+By having multiple attributes of the same type we're effectively creating a 
+[multivalued attribute](04-write.md#multivalued-attributes) (i.e. having an attribute type instantiated with multiple 
+values).
+
+Use the `@key` keyword to limit the cardinality to exactly **one** and add **uniqueness** constraint. 
+See example in [Key attribute](#define-attribute-types) section.
+
+##### Plays a role
+
+To add roles that entities of a specific entity type can play, use the `plays` keyword.
+
+<!-- test-ignore -->
+```typeql
+define
+
+access sub relation, relates accessed-object;
+object sub entity, abstract, plays access:accessed-object;
+```
+
+##### Subtypes another entity
+
+<div class="note">
+[Note]
+All types that are subtyping `entity` base type directly or through other subtypes are called entity types. 
+Instances of these types are called entities. The same approach can be applied to attributes and relations.
+</div>
+
+An entity type can subtype another entity type by using the same `sub` keyword, but replacing the `entity` after it 
+with the name of another entity type to subtype.
+
+<!-- test-ignore -->
+```typeql
+define
+
+path sub attribute, value string;
+
+object sub entity, abstract, owns object-type, plays access:accessed-object;
+resource sub object, abstract;
+file sub resource, owns path;
+```
+
+In the above example, the `resource` and `file` entity types are subtypes of the `object`, which itself is a subtype 
+of the `entity` base type. They inherit the `object-type` attribute type ownership from it as well as its 
+`access:accessed-object` role. However, while the `resource` subtype is abstract, the `file` subtype is not. Hence, we 
+can create `file` entities, but not `resource` entities.
+
+Further, the `path` attribute type will only be owned by the `file` entity type and any other entity types which 
+subtype it or directly define ownership.
+
+##### Overrides inherited attribute ownership
+
+To override an inherited ownership use `owns` keyword with the new attribute name, followed by the `as` keyword and 
+the inherited attribute type name. An attribute type with overridden ownership is defined in the schema as the 
+subtype of an inherited attribute type. Hence, an inherited attribute type is abstract and has the same value type 
+as the new attribute type.
```

Comment:
This section isn't clear to me as it mixes attribute inheritance and ownership inheritance.